repo/packages/H/hdfs/500/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "2.4.0-2.6.0-cdh5.11.0"
+  ],
+  "downgradesTo": [
+    "{{downgrades-to}}"
```

Comment:
Why isn't `{{downgrades-to}}` rendered?

Why isn't this `2.4.0-2.6.0-cdh5.11.0`? 

Or if we can't downgrade once we upgrade to `2.5.0-2.6.0-cdh5.16.1`, shouldn't this be set to its own version - i.e., `2.5.0-2.6.0-cdh5.16.1`?